var/spack/repos/builtin/packages/care/package.py

```diff
@@ -3,42 +3,59 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import socket
+
 from spack.package import *
 
+from .blt import llnl_link_helpers
+
 
-class Care(CMakePackage, CudaPackage, ROCmPackage):
+class Care(CachedCMakePackage, CudaPackage, ROCmPackage):
     """
-    Algorithms for chai managed arrays.
+    CHAI and RAJA extensions (includes data structures and algorithms).
     """
 
     homepage = "https://github.com/LLNL/CARE"
     git = "https://github.com/LLNL/CARE.git"
     tags = ["radiuss"]
 
-    license("GPL-2.0-or-later")
+    license("BSD-3-Clause")
 
-    maintainers("adayton1")
+    maintainers("adayton1", "adrienbernede")
 
-    version("develop", branch="develop", submodules="True")
-    version("master", branch="main", submodules="True")
+    version("develop", branch="develop", submodules=False)
+    version("master", branch="main", submodules=False)
```

Comment:
It looks like the branch is actually `master`, not `main` (and there is no `main` branch ATM).

Reply:
Done